src/disk_io_thread.cpp

```diff
@@ -2528,8 +2521,13 @@ namespace libtorrent
 		if (rd == nullptr) rd = &tmp;
 
 		std::unique_ptr<std::vector<std::string>> links(j->d.links);
+		bool no_recheck_incomplete_resume
```

Comment:
this bool should be const.. and, I wonder if there's a better way to pass this in. just a bool feels perhaps a bit untyped